src/main/java/com/google/sps/servlets/NewPostServlet.java

```diff
@@ -18,12 +18,19 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
+
+/* 
```

Comment:
Make it a doc comment. javadocs are a specific type of comment but the distinction is important, automated systems can read and expose them differently. (literally just add another * to the opening)

https://www.tutorialspoint.com/java/java_documentation.htm